1-js/13-modules/01-modules-intro/article.md

```diff
@@ -229,162 +229,163 @@ Compare it to non-module scripts, where `this` is a global object:
+4. در حین عملیات، تبدیل ها و ارتقاهای دیگری هم ممکن است انجام شود:
+    - کدهایی که هیچ وقت اجرا نمی شوند، حذف می شوند.
+    - exportهایی که استفاده نمی شوند، پاک می شوند.("tree-shaking").
+    - عباراتی که مخصوص زمان توسعه نرم افزار هستند مانند `console` و `debugger` حذف می شوند.
+    - سینتکس و املای مدرن جاوااسکریپت ممکن است به نمونه های قدیمی با عملکرد مشابه توسط [Babel](https://babeljs.io/) تبدیل شوند.
+    - فایل نهایی فشرده می شود. (فاصله های پاک می شوند، متغیرها با نام های کوتاه تر جایگزین می شوند و غیره)
 
-If we use bundle tools, then as scripts are bundled together into a single file (or few files), `import/export` statements inside those scripts are replaced by special bundler functions. So the resulting "bundled" script does not contain any `import/export`, it doesn't require `type="module"`, and we can put it into a regular script:
+اگر ما از ابزارهای باندل استفاده کنیم، در این صورت تمام اسکریپت ها با هم در یک فایل ( یا تعداد کمی فایل ) جمع می شوند، عبارات `import/export` داخل اسکریپت ها با توابع خاص باندلر ها جایگزین می شوند. در نتیجه اسکریپت باندل نهایی هیچ عبارت `import/export` ندارد، این اسکریپت نیازی به `type="module"` ندارد، و ما می توانیم آن را در یک اسکریپت معمولی بگذاریم.
 
 ```html
-<!-- Assuming we got bundle.js from a tool like Webpack -->
+<!-- با فرض اینکه ما bundle.js را از یک ابزار مانند Webpack گرفته ایم -->
 <script src="bundle.js"></script>
 ```
 
-That said, native modules are also usable. So we won't be using Webpack here: you can configure it later.
+با این حساب، ماژول ها بومی و نیتیو هم قابل استفاده هستند. در نتیجه ما از Webpack در اینجا استفاده نمی کنیم: شما می توانید آن را در آینده تنظیم کنید.
 
-## Summary
+## خلاصه
 
-To summarize, the core concepts are:
+برای خلاصه کردن، مفاهیم اصلی اینها هستند:
 
-1. A module is a file. To make `import/export` work, browsers need `<script type="module">`. Modules have several differences:
-    - Deferred by default.
-    - Async works on inline scripts.
-    - To load external scripts from another origin (domain/protocol/port), CORS headers are needed.
-    - Duplicate external scripts are ignored.
-2. Modules have their own, local top-level scope and interchange functionality via `import/export`.
+1. هر ماژول یک فایل است. برای اینکه عبارات `import/export` کار بکنند، مرورگرها نیاز به `<script type="module">` دارند. ماژول های چندین تفاوت با اسکریپت های معمولی دارند:
+    - به صورت پیش فرض به تعویق افتاده (Deferred) هستند.
+    - در اسکریپت های inline Async جواب می دهد.
+    - برای بارگزاری اسکریپت های خارجی(external) از منابع دیگر (دامنه/پروتکل/پورت)، هدر های CORS نیاز هستند.
+    - اسکریپت های مشابه external نادیده گرفته می شوند.
+2. ماژول های اسکوپ سطح بالای خود را دارند و از طریق عبارات ‍`import/export` کارایی های خود را با دیگر اسکریپت های به اشتراک می گذارند.
 3. Modules always `use strict`.
```

Comment:
Please remove the English term after translating it!

Reply:
sorry i will check whole translate again.

Reply:
It happens. :) Although I think this was the last one. 🤔